models/edr/dbt_artifacts/dbt_exposures.sql

```diff
@@ -1,8 +1,8 @@
 {{
   config(
-    materialized='table',
+    materialized='incremental',
```

Comment:
I checked, and the default strategy for all adapters is either append or merge - which means it should be safe concurrently